src/components/maincontroller.ts

```diff
@@ -509,23 +484,32 @@ export function shuffle(
 ): Promise<void> {
     return getShuffleItems(data.userId, item).then(function (result) {
         options.items = result.Items;
-        playbackMgr.playFromOptions(data.options);
+        PlaybackManager.playFromOptions(data.options);
     });
 }
 
+/**
+ * This function fetches the full information for an item before playing it.
+ * The provided item is not complete.
+ *
+ * Old behavior: The original properties would be copied over to the fetched one,
+ * but just the fetched item should be fine
```

Comment:
I think this is not clear enough, I don't understand it for instance 😅.

Reply:
will make clearer